unstructured/documents/elements.py

```diff
@@ -454,9 +454,11 @@ def __init__(
         coordinate_system: Optional[CoordinateSystem] = None,
         metadata: Optional[ElementMetadata] = None,
         detection_origin: Optional[str] = None,
+        embeddings: Optional[List[float]] = None,
     ):
         metadata = metadata if metadata else ElementMetadata()
         self.text: str = text
+        self.embeddings: Optional[str] = embeddings
```

Comment:
Why is the typing here set to `Optional[str]`?

Reply:
Typo

Reply:
```suggestion
        self.embeddings: Optional[List[float]] = embeddings
```